packages/terra-content-container/tests/jest/__snapshots__/ContentContainer.test.jsx.snap

```diff
@@ -73,5 +106,8 @@ exports[`should render content 1`] = `
       </p>
     </div>
   </div>
+  <div
+    className="footer"
+  />
 </div>
 `;
```

Comment:
We should add an example using the footer into the example pages.

https://github.com/cerner/terra-core/tree/master/packages/terra-site/src/examples/content-container
http://engineering.cerner.com/terra-core/#/site/content-container